features/so_adj_ratio.py

```diff
@@ -0,0 +1,22 @@
+import re
+import string
+
+def CountStartleFeatures_SoAdj(text):
+    text = re.sub('['+string.punctuation+']', '', text)    #removing punctuation from text
```

Comment:
No need to remove puctuation and casting to lowercase. The text is already stored and processed normalized. Please remove this stuff.